specification/containerservices/resource-manager/Microsoft.ContainerService/stable/2018-03-31/managedClusters.json

```diff
@@ -231,6 +231,96 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}/listClusterAdminCredential": {
+      "post": {
+        "tags": [
+          "ManagedClusters"
+        ],
+        "operationId": "ManagedClusters_list_clusteradmin_credential",
```

Comment:
Could you rename this to "ManagedClusters_ListClusterAdminCredential"? That seems to be the convention in swagger here.

Reply:
From the `CredentialResults` return value, it appears there could be multiple name+value credential pairs returned for a clusteradmin. Is that correct? If so, let's name this "ManagedClusters_ListClusterAdminCredential**s**".

Reply:
yes, it could be multiple pairs. I will rename it to "ManagedClusters_ListClusterAdminCredentials".